pages/reference/supervisor/bandwidth-reduction.md

```diff
@@ -4,52 +4,47 @@ title: Reduce bandwidth usage
 # Reduce bandwidth usage
 
-In order to provide users with essential information about their devices, the {{ $names.company.lower }} Supervisor on the device regularly keeps the API informed about the device's state. Upon receiving a target device state which differs from the current device state, the Supervisor applies changes, such as downloading new application updates or setting environment variables. While a quick response to actions is great during development, it also leads to of increased data usage.
+balenaOS and the device Supervisor make it easy to manage a device remotely with features like forwarding container logs, hardware metrics reporting, and Cloudlink for on-demand external access. However these features consume network bandwidth, and typically cellular providers  charge by the megabyte. So to trade off between cost and features, you may configure their use individually.
 
-In order to give our power users control over data flow, we enabled a few `BALENA_` variables in the [configuration][configuration] tab.
+To help you understand the costs, we measured data use over a cellular connection for features that require network transfer. **The minimum bandwidth required is approximately 1 MB per month** for balenaOS v2.108.18 and Supervisor v14.4.4, when all of the features in the table below are turned off and API Polling is set to the minimum value of once per day.
 
-| Variable                               | Allowed Value      | Action                                             | Default | Available with Supervisor Version |
-|----------------------------------------|--------------------|----------------------------------------------------|---------|-----------------------------------|
-| `BALENA_SUPERVISOR_VPN_CONTROL`        | true/false         | Enable / Disable cloudlink                         | true    | v1.1.0                            |
-| `BALENA_SUPERVISOR_CONNECTIVITY_CHECK` | true/false         | Enable / Disable cloudlink connectivity check      | true    | v1.3.0                            |
-| `BALENA_SUPERVISOR_POLL_INTERVAL`      | 600000 to 86400000 | API Poll interval in milliseconds                  | 900000  | v1.3.0                            |
-| `BALENA_SUPERVISOR_LOG_CONTROL`        | true/false         | Enable / Disable logs from being sent to API       | true    | v1.3.0                            |
-| `BALENA_SUPERVISOR_HARDWARE_METRICS`   | true/false         | Enable / Disable device hardware metrics reporting | true    | v12.8.0                           |
+You may control these features from the fleet/device configuration tabs on the dashboard and as device variables. See the dashboard [docs][device-configuration] and the Configuration List [page][configuration-list]. Below the table we describe our measurement tool so you can repeat the tests.  Finally, be sure to read the [Side Effects](#side-effects--warnings) section for the impact of turning off each feature, particularly Cloudlink VPN.
```

Comment:
```suggestion
You may control these features from the fleet/device configuration tabs on the dashboard and as device variables. See the dashboard [docs][device-configuration] and the Configuration List [page][configuration-list]. Below the table we describe our measurement tool so you can repeat the tests.  Finally, be sure to read the [Side Effects](#side-effects--warnings) section for the impact of turning off each feature, particularly Cloudlink.
```

Reply:
Thanks @dfunckt , I have removed "VPN" from references to Cloudlink.